app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,53 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+public class ProductService {
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+
+    /**
+     * product 객체를 리포지토리에 생성한다
+     * @param product 생성할 프러덕트
+     * @return 생성한 프러덕트
+     */
+    public Product createProduct(Product product) {
+        return productRepository.save(product);
+    }
+
+    /**
+     * 인자로 받은 id 값을 가진 프러덕트를 찾는다
+     * @param productId 프러덕트 id
+     * @return 찾은 프러덕트
+     */
```

Comment:
이렇게 해도 괜찮겠죠.
```suggestion
    /**
     * 프러덕트를 찾아 리턴한다.
     * @param productId 프러덕트 id
     * @return 찾은 프러덕트
     */
```

JavaDoc에서 `~을 리턴합니다.` 처럼 표현하는 것은 Java 스탠다드 라이브러리 스타일이기도 합니다. 다음은 String의 `charAt` 메소드에요.

![image](https://user-images.githubusercontent.com/1855714/176689054-63a60bab-02ac-4d6d-8cf9-04bfa5df3833.png)